Issutracker : #iXXXXX#
CWS         : N/A
Author      : NAKATA Maho <maho@openoffice.org> (JCA)
Description :
in odk project, machine type is determined by config.guess,
for FreeBSD, it returns freebsd5.3 or freebsd4.11 etc.

Index: odk/setsdkenv_unix.sh.in
===================================================================
RCS file: /cvs/api/odk/setsdkenv_unix.sh.in,v
retrieving revision 1.5
diff -u -r1.5 setsdkenv_unix.sh.in
--- odk/setsdkenv_unix.sh.in	29 Mar 2005 12:03:27 -0000	1.5
+++ odk/setsdkenv_unix.sh.in	15 May 2005 09:06:22 -0000
@@ -130,7 +130,7 @@
     export LD_LIBRARY_PATH
     ;;
 
-  freebsd)
+  freebsd*)
     if [ -n "$OO_SDK_CPP_HOME" ]
     then
 	SDK_GXX_INCLUDE_PATH=`echo "#include <cstring>" | $OO_SDK_CPP_HOME/g++ -E -xc++ - | sed -n '/.*1*"\(.*\)\/cstring".*/s//\1/p' | head -n 1`
Index: odk/setsdkenv_unix.csh.in
===================================================================
RCS file: /cvs/api/odk/setsdkenv_unix.csh.in,v
retrieving revision 1.5
diff -u -r1.5 setsdkenv_unix.csh.in
--- odk/setsdkenv_unix.csh.in	29 Mar 2005 12:03:14 -0000	1.5
+++ odk/setsdkenv_unix.csh.in	15 May 2005 09:06:38 -0000
@@ -114,7 +114,7 @@
     endif
   breaksw
 
-  case "freebsd":
+  case "freebsd*":
     if ( $?OO_SDK_CPP_HOME == 0) then
 	set sdk_gxx_include_path=`echo "#include <cstring>" | ${OO_SDK_CPP_HOME}/g++ -E -xc++ - | sed -n '/.*1*"\(.*\)\/cstring".*/s//\1/p' | head -n 1`
 	setenv SDK_GXX_INCLUDE_PATH ${sdk_gxx_include_path}
